pycbc/inject/injfilterrejector.py

```diff
@@ -150,11 +168,17 @@ class InjFilterRejector(object):
 
     def __init__(self, injection_file, chirp_time_window,
                  match_threshold, f_lower, coarsematch_deltaf=1.,
-                 coarsematch_fmax=256, seg_buffer=10):
+                 coarsematch_fmax=256, seg_buffer=10, inj_trigger_window=None):
         """Initialise InjFilterRejector instance."""
         # Determine if InjFilterRejector is to be enabled
-        if injection_file is None or injection_file == 'False' or\
-                (chirp_time_window is None and match_threshold is None):
+        if (
```

Comment:
The "else" branch of this if statement is unnecessary, please remove.

Reply:
Done (will be pushed later with other fixes)